content/hands-on-exposed-thing-guide.md

```diff
@@ -164,7 +165,9 @@ Resulting from the discussions in the Scripting API, the API has changed on 28.1
 
 You can see a git diff [here](https://github.com/tum-esi/wot-sys/commit/6cef8530b3317d98c2a7dea389c92ba2786be892#diff-0d33955cb472f41f07397fc2687c6425R10)
 
-
 ## Change from Version 0.7.X to 0.8.X for Exposed Things
 
-<mark>TODO</mark>
+Version 0.8.X handles properties and interaction inputs differently than previous versions. Below is the list of changes:
+
+- Properties are not handled internally by node-wot anymore. Property handlers need to read and write to variables or class members that are explicitly defined in the code for the Exposed Things
+- The value of interaction inputs cannot be accessed directly. `WoT.InteractionInput` objects have a member function `value()` that returns a promise that will resolve to the value of the input when available. You can either use `input.value().then(...)` or `await input.value()` in an `async` function.
```

Comment:
@egekorkan argues that 
"Change from Version 0.7.X to 0.8.X"
should come before 
"Change from Version 0.6.X to 0.7.X"

I am fine either way but maybe its more obvious to put it in reverse order

Reply:
I agree, will do

Reply:
Please let me know whether you find the time to update the PR. In case I can do the re-ordering also...

Reply:
I reordered the changes section